sources/components/custom-styles/template-cs-content.scss

```diff
@@ -555,5 +555,195 @@
   */
   #{--content-links-hover-color}: $color-two;
 
+  /* VoogStyle
+    "pathI18n": ["product", "title"],
+    "titleI18n": "size",
+    "editor": "rangePicker",
+    "min": 8,
+    "max": 100,
+    "unit": "px",
+    "scope": "global"
+  */
+  #{--content-area-product-title__font-size}: $h1-size;
```

Comment:
Regular product page seems to use `font-size: 28px` for title, not regular `h1` font size
https://github.com/Voog/design-rotterdam/blob/8f8685555aff694d8120202ebb069c60d345b25e/sources/stylesheets/_product.scss#L264

Reply:
Ok. Let's legacy "product" layout remains untouched at the moment.